### DISCUSSION TITLE
Is this normal that systemd tell me floorp register object KeyboardEvent ?

### DISCUSSION THREAD

**Discussion by bouteillerAlan** [Floorp-Projects/Floorp#1281] [Q&A] [answered]
Whenever I type in floorp I have this log in `journalctl` (tested with firefox with the same plugin and I have no log) : 
![Screenshot_20240723_155147](https://github.com/user-attachments/assets/bdc1b068-3529-4a02-97d5-e09f8f0a0903)

**Comment by bouteillerAlan** [comment]
Any update on that?

**Reply by regularentropy** [reply, marked as answer]
Yes, it's a debug log.

**Reply by bouteillerAlan** [reply]
Ok but if this debug logs password this is a security issue no?

**Reply by regularentropy** [reply]
> Ok but if this debug logs password this is a security issue no?

It can't possibly print out the password.
